plugins/modules/site_intent.py

```diff
@@ -1090,6 +1078,47 @@ def main():
         if config_verify:
             dnac_site.verify_diff_state_apply[state](config).check_return_status()
 
+    if dnac_site.created_site_list and dnac_site.updated_site_list:
```

Comment:
can we write an API update_site_messages and call it? 
def update_site_messages(dnac_site):